src/matcha_ml/core/core.py

```diff
@@ -120,7 +135,7 @@ def analytics_opt_in() -> None:
 
 
 def remove_state_lock() -> None:
-    """Unlock remote state."""
+    """Unlock the remote state."""
```

Comment:
Can we add a sentence about when/why to use this and maybe add a warning about possibly breaking the state file if it is used?